src/Components/MovieDetailsPage/MovieDetails.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import "./moviedetailspage.css";
```

Comment:
Same thing here- there could be separate css files for all js files. Much cleaner to read.